ikarus/io/resultfunction.hh

```diff
@@ -50,7 +50,7 @@ class ResultFunction : public Dune::VTKFunction<typename FE::GridView>
 {
 public:
   using FiniteElement          = FE;
-  using FERequirementType      = typename FiniteElement::FERequirementType;
+  using FERequirementType      = typename FiniteElement::Requirement;
```

Comment:
To stay consistent, remove Type
```suggestion
  using Requirement     = typename FiniteElement::Requirement;
```

Reply:
I thought it is more consistent that only the FEs tell their type by Requirement alone. All other classes (assemblers, resultfunctions) export it as FERequirementType, since it is not the requirement of these classes.

Reply:
Unfortunately not since then we have a name class with the actual class FERequirements.

Reply:
at least that was the reason I added it in the first place